Fails to remove gitlab deployment

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/woodpecker-ci/plugin-deployments/forge.(*Gitlab).RemoveDeployment(0xc000214410, {0xbc9678, 0xc0000c7680}, {{0xc000000222, 0x2}, {0xc000000055, 0xb}, {0xc00000006e, 0x5}, {0xc00002a10c, ...}, ...}, ...)
	/src/forge/gitlab.go:149 +0x4e5
main.(*Plugin).execute(0xc0000b04b0, {0xbc9678, 0xc0000c7680})
	/src/plugin.go:67 +0x5ce
codeberg.org/woodpecker-plugins/go-plugin.(*Plugin).action(0xc0000f7b88, {0xbc9678, 0xc0000c7680}, 0xc000128008)
	/go/pkg/mod/codeberg.org/woodpecker-plugins/go-plugin@v0.6.0/plugin.go:96 +0x12c
github.com/urfave/cli/v3.(*Command).Run(0xc000128008, {0xbc9598, 0xfb8ce0}, {0xc0000aa0a0, 0x1, 0x1})
	/go/pkg/mod/github.com/urfave/cli/v3@v3.0.0-alpha9.0.20241004184838-20ef97b2155a/command.go:660 +0x19e8
codeberg.org/woodpecker-plugins/go-plugin.(*Plugin).Run(0xb0c496?)
	/go/pkg/mod/codeberg.org/woodpecker-plugins/go-plugin@v0.6.0/plugin.go:106 +0x3a
main.main()
	/src/main.go:43 +0x9ab
```

--- a/forge/gitlab.go
+++ b/forge/gitlab.go
@@ -145,11 +145,11 @@ func (g *Gitlab) RemoveDeployment(ctx context.Context, repo plugin.Repository, n
 			return err
 		}
 
-		// TODO: delete all deployments
-		deployment := deployments[0]
-		_, err = g.Deployments.DeleteProjectDeployment(repoID, deployment.ID, gitlab.WithContext(ctx))
-		if err != nil {
-			return err
+		for _, deployment := range deployments {
+			_, err = g.Deployments.DeleteProjectDeployment(repoID, deployment.ID, gitlab.WithContext(ctx))
+			if err != nil {
+				return err
+			}
 		}
 	}
 

--- a/forge/gitlab.go
+++ b/forge/gitlab.go
@@ -137,20 +137,6 @@ func (g *Gitlab) RemoveDeployment(ctx context.Context, repo plugin.Repository, n
 		if err != nil {
 			return err
 		}
-
-		deployments, _, err := g.Deployments.ListProjectDeployments(repoID, &gitlab.ListProjectDeploymentsOptions{
-			Environment: gitlab.Ptr(name),
-		}, gitlab.WithContext(ctx))
-		if err != nil {
-			return err
-		}
-
-		for _, deployment := range deployments {
-			_, err = g.Deployments.DeleteProjectDeployment(repoID, deployment.ID, gitlab.WithContext(ctx))
-			if err != nil {
-				return err
-			}
-		}
 	}
 
 	return nil

